src/components/PendingNwc.tsx

```diff
@@ -78,8 +78,19 @@ export function PendingNwc() {
             await state.mutiny_wallet?.approve_invoice(item.id);
             await vibrateSuccess();
         } catch (e) {
-            setError(eify(e));
-            console.error(e);
+            // If we've already paid this invoice, just ignore the error
+            // we just want to remove it from the list and continue
+            if (e === "An invoice must not get payed twice.") {
```

Comment:
it's usually `e.message`, not the `e` object itself. is there a way we can test this?

Reply:
hard to test, this is a pretty uncommon edge case